### ISSUE TITLE
Decorative images are flagged as images with missing alt text

### ISSUE THREAD

**Issue by wenche** [equinor/energyvision#754]
Tool: Sitebulb
Page from the screenshots: https://energyvision.app.radix.equinor.com/sustainability/environment

![Skjermbilde 2022-01-19 kl. 09.50.55.png](https://images.zenhubusercontent.com/5fb7aa9b653d5135bb8c50f5/f1f6e55e-156d-4313-9b18-3e5d40612aeb)![Skjermbilde 2022-01-19 kl. 09.51.04.png](https://images.zenhubusercontent.com/5fb7aa9b653d5135bb8c50f5/170db1e3-4b45-442a-b213-eb9a8a5aba89)

**Comment by SvSven** [comment]
I recall there was a difference between how Chrome and other browsers handled an empty alt tag, which may be causing this with sitebulb.

Not sure if there is anything we can do here since the guidelines states that we are supposed to write it like `<img src="blabla" alt="" />` - which is what we do.

Supposedly an `aria-hidden` attribute is not needed in these cases, but maybe adding that will at least silence the warnings?

**Comment by SvSven** [comment]
https://www.w3.org/WAI/tutorials/images/decorative/

**Comment by nilsml** [comment]
Browser specific?

**Comment by nilsml** [comment]
Run Lighthouse and check

**Comment by wenche** [comment]
I might actually have looked at the wrong image. Looks like it is the teaser image, seems like we don't add an empty alt text here for decorative img
![Skjermbilde 2022-01-20 kl. 08.18.28.png](https://images.zenhubusercontent.com/5fb7aa9b653d5135bb8c50f5/da43b241-445f-461f-b8a0-69a2817b05a1)
![Skjermbilde 2022-01-20 kl. 08.19.33.png](https://images.zenhubusercontent.com/5fb7aa9b653d5135bb8c50f5/dff65d88-9f8e-4d47-ba64-04977ba8ef55)

**Comment by adneforslund** [comment]
@SvSven review please

**Comment by SvSven** [comment]
Shows up as an alt tag without value in Edge

![image.png](https://images.zenhubusercontent.com/5dba9d22e2edb600015fe6c0/86b93ce0-a8fc-48a8-948c-1b497744fa97)

And an alt with empty value in FireFox

![image.png](https://images.zenhubusercontent.com/5dba9d22e2edb600015fe6c0/b3a30fc9-925e-4dbe-8b72-fa4c713e9ecf)

Which is in line with what I have seen before, but that may have been with other images - in which case based on @wenche 's message this should now be fixed then?

**Comment by wenche** [comment]
Let's hope so! We can verify next time Siteimprove crawls

**Comment by wenche** [comment]
Next crawl is scheduled by tomorrow

**Comment by wenche** [comment]
Sitebulb is now happy :thumbsup: 

Still some issues with missing alt text, but that's for the AEM content.

![Skjermbilde 2022-02-02 kl. 08.26.51.png](https://images.zenhubusercontent.com/5fb7aa9b653d5135bb8c50f5/c7ab7d2d-47ae-4c0b-bda7-6bb760338e55)
